build.gradle.kts

```diff
@@ -126,6 +126,9 @@ tasks {
     }
 
     wrapper {
-        gradleVersion = "6.0.1"
+        gradleVersion = "6.1.1"
+        distributionType = Wrapper.DistributionType.ALL
     }
+    register("noOp")
```

Comment:
What's this?

Reply:
I mentioned that in the other PR: https://github.com/jeremymailen/kotlinter-gradle/pull/129#discussion_r370923114
When invoking `./gradlew` without arguments it invokes `:help` task by default. This line (and the one below) sets noOp task to be invoked on every empty gradle invocation. 
But yeah, that's just _my preference_ 😛 

🤔 Although,  just realized I should be able to move that to my local Gradle init script instead! So I'll remove that.